mealplanner-ui/src/pages/MealPlans/CreateMealPlan.tsx

```diff
@@ -102,139 +111,164 @@ export const CreateMealPlan = ({ connection, refetch }: { connection: string, re
+          >
+            <FormControlLabel
+              value="mealPlan"
+              control={<Radio checked={planType === 'mealPlan'}/>}
+              label="Create New Meal Plan"
+            />
+            <FormControlLabel
+              value="template"
+              control={<Radio checked={planType === 'template'}/>}
+              label="Create New Template"
+            />
+          </RadioGroup>
+        </FormControl>
+        </Grid>
+        </DialogTitle>
+         
+            <DialogContent>
+              <Grid container columns={6} spacing={2}>
+              {planType === "mealPlan" && (
+                <Grid item xs={6}>
+                  <Autocomplete
+                    options={allUsers || []}
+                    renderInput={(params) => (
+                      <TextField
+                        {...params}
+                        label="Assign user"
+                        id="user"
+                        variant="filled"
+                      />
+                    )}
+                    onChange={(e, value) => {
+                      setUserId(value?.rowId);
+                    }}
+                  ></Autocomplete>
+                </Grid>
                 )}
-                onChange={(e, value) => {
-                  setUserId(value?.rowId);
-                }}
-              ></Autocomplete>
-            </Grid>
-            <Grid item xs={3}>
-              <Tooltip arrow placement="bottom-start" title={titleEn}>
-                <TextField
-                  id="nameEn"
-                  label="Meal Plan Name*"
-                  autoFocus
-                  margin="dense"
-                  variant="standard"
-                  fullWidth
-                  onChange={(e) => {
-                    setNameEn(e.target.value);
-                  }}
-                />
-              </Tooltip>
-            </Grid>
-            <Grid item xs={3}>
-              <Tooltip arrow placement="bottom-start" title={titleFr}>
-                <TextField
-                  id="nameFr"
-                  label="nom du plan de repas"
-                  margin="dense"
-                  variant="standard"
-                  fullWidth
-                  onChange={(e) => {
-                    setNameFr(e.target.value);
-                  }}
-                />
-              </Tooltip>
-            </Grid>
-            <Grid item xs={3}>
-              <Tooltip arrow placement="bottom-start" title={titleDescEng}>
-                <TextField
-                  id="descriptionEn"
-                  label="Description"
-                  multiline
-                  variant="filled"
-                  fullWidth
-                  onChange={(e) => {
-                    setDescriptionEn(e.target.value);
-                  }}
-                />
-              </Tooltip>
-            </Grid>
-            <Grid item xs={3}>
-              <Tooltip arrow placement="bottom-start" title={titleDescFr}>
```

Comment:
Looks like description Tooltip is also missing.

Reply:
Thanks for pointing this out. Fixed it and attached screenshots.